companies/general-motors.md

```diff
@@ -0,0 +1,10 @@
+---
+fortune_rank: 18
+name: General Motors
+slug: general motors

```

Comment:
```suggestion
slug: general-motors

```

Reply:
@Nimanita One tiny update and then I will merge :) Would you like to commit the suggestion here or push an update for the slug?

Reply:
I will commit the suggestion.It will be easy for me.